templates/core/terraform/firewall/firewall.tf

```diff
@@ -216,7 +216,8 @@ resource "azurerm_firewall_network_rule_collection" "resource_processor_subnet"
       "AzureResourceManager",
       "AzureContainerRegistry",
       "AzureMonitor",
-      "Storage"
+      "Storage",
+      "AzureKeyVault"
```

Comment:
Please add this to the https://github.com/microsoft/AzureTRE/blob/develop/resource_processor/vmss_porter/readme.md#network-requirements

Reply:
Done